cowprotocol/cowswap
# [ACCOUNT] Split pages

# Summary

Closes #564 

<img width="1283" alt="image" src="https://user-images.githubusercontent.com/21335563/180742283-810ac22d-e518-4319-a375-9799e4613fa7.png">

## Testing
1. head to account page
2. click links
3. check each page

**Note:** Wasn't able to test vCOW balance/lGNO in dev, will try in pr link
